tests/conftest.py

```diff
@@ -190,15 +190,26 @@ def test_account():
 ########## DEFAULT LEVELS ##########
 
 default_fixtures = {}
-default_fixtures[TestLevels.UNIT] = {"cluster": [local_docker_cluster_public_key]}
+default_fixtures[TestLevels.UNIT] = {
+    "cluster": [
+        local_docker_cluster_public_key_logged_in,
+        local_docker_cluster_public_key_logged_out,
```

Comment:
I had only included one fixture in UNIT as a sort of smoke test for the core cluster behavior, and rely on LOCAL to properly test behavior across usage modes. That's not exactly a rigorous line. Do you have a particular thought about where to draw the line between UNIT and LOCAL such that these should both be in UNIT?